### ISSUE TITLE
Only InModule tasks are available

### ISSUE THREAD

**Issue by johnLegasse** [cesarferreira/gradlr#6]
My main project is composed by 3 modules and 1 lib, when I launch gradlr and want to perform a clean, Gradlr only propose module1:clean -> moduleX:clean and lib1:clean
The project tasks are missing.

**Comment by cesarferreira** [comment]
I released a new version a few days ago that has a better parser of the tasks, maybe you didn't updated yet, can you check please?

**Comment by cesarferreira** [comment]
everything is fixed on version `1.2.0`

**Comment by johnLegasse** [comment]
Hey just checked 1.2.0, it seems to be working for primitive task (clean, buildNeeded etc).

Flavors etc are not listed at root level. For example 
"gradlr assembleDebug" will show :
```
Module1:assembleDebug
Module2:assembleDebug
Lib:assembleDebug
Module3:assembleDebug
```

While "gradlr clean" will show :
```
Module1:clean
Module2:clean
Lib:clean
Module3:clean
clean
```

Also I don't know you but personally I tend to use the project task way more often than the module task. Would that make sense to sort the task to put the project task on top (clean, module1:clean etc)?
